gene2phenotype_project/gene2phenotype_app/serializers.py

```diff
@@ -69,7 +68,26 @@ def create_stable_id():
 
         return stable_id_instance
     
-    
+    def update_g2p_id_status(self, is_live):
+        """
+            Update the status of the G2P stable id.
+            Set 'is_live' to:
+                0: entry is not published (live)
+                OR
+                1: entry is published
```

Comment:
```suggestion
                0: entry is not published
                OR
                1: entry is published (live)
```

Reply:
the comment was a bit confusing so suggested this minor change to remove any confusion